tests/hazmat/backends/test_openssl.py

```diff
@@ -529,7 +534,7 @@ def test_checks_for_unsupported_extensions(self):
         ).not_valid_after(
             datetime.datetime(2020, 1, 1)
         ).add_extension(
-            x509.IssuerAlternativeName([x509.DNSName(u"crypto.io")]), False
+            DummyExtension(), False
```

Comment:
That test is for CSRBuilder. Removing this test results in an uncovered line (just checked locally). I'll move this test into test_x509 though since it's not really backend specific.